test/cluster/rollout.sh

```diff
@@ -25,7 +25,13 @@ os::cmd::expect_success "oc project logging"
 os::log::info "Checking for DeploymentConfigurations..."
 for deploymentconfig in ${OAL_EXPECTED_DEPLOYMENTCONFIGS}; do
 	os::cmd::expect_success "oc get deploymentconfig ${deploymentconfig}"
-	os::cmd::expect_success "oc rollout status deploymentconfig/${deploymentconfig}"
+
+# this is to get around the current kubelet flake where we need to re-rollout to get the dc running
+#  for sanity sake and to not get stuck in the case of real issues, we will only do this once per dc
+	if [[ ! $(oc rollout status deploymentconfig/${deploymentconfig}) ]]; then
+		os::cmd::expect_success "oc rollout latest ${deploymentconfig}"
+		os::cmd::expect_success "oc rollout status deploymentconfig/${deploymentconfig}"
```

Comment:
if you want to wait for the deployment to be completed use `status -w`

Reply:
when i tested this locally with a fresh rollout it waited with the above command... i'll watch the job though to make sure

Reply:
hmm
```
# oc rollout status --help
....
  -w, --watch=true: Watch the status of the rollout until it's done.
# oc version
oc v3.9.0-alpha.4+e28a3b2-439
```